ui/app/store/actions.js

```diff
@@ -2494,9 +2494,9 @@ export function rejectPendingApproval(id, error) {
     await promisifiedBackground.rejectPendingApproval(id, error);
     // Before closing the current window, check if any additional confirmations
     // are added as a result of this confirmation being rejected
-    const { pendingApprovals } = await promisifiedBackground.getState();
+    const { pendingApprovals } = await forceUpdateMetamaskState(dispatch);
     if (Object.values(pendingApprovals).length === 0) {
-      dispatch(closeCurrentNotificationWindow());
+      dispatch(closeNotificationWindow());
```

Comment:
What does this function do :thinking:

Reply:
haha when i read it i was like "doh i used the wrong function" -- i added a commit to remove the dead code.